frontend/src/lib/components/instance-views/views/views/openai-chat-markdown/SystemBlock.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+	export let input: string;
+</script>
+
+<p class="m-0">
+	<b>System:</b>
```

Comment:
Is there a reason to not render the system message in markdown? It might be simpler to just render all messages.

Reply:
Not sure tbh, just copied over basically from [here](https://github.com/zeno-ml/instance-views/blob/main/openai-chat-markdown/src/SystemBlock.svelte). Maybe @cabreraalex had a reason not to render markdown for the system message?

Reply:
nope just think they wouldn't be markdown, maybe go for it